src/applications/personalization/common/mocks/feature-toggles/index.js

```diff
@@ -1,64 +1,59 @@
-const generateFeatureToggles = (toggles = {}) => {
-  const {
-    authExpVbaDowntimeMessage = true,
-    myVaEnableNotificationComponent = true,
-    myVaHideNotificationsSection = true,
-    myVaUseExperimental = true,
-    myVaUseExperimentalFrontend = true,
-    myVaUseExperimentalFullstack = true,
-    myVaNotificationDotIndicator = true,
-    myVaEnableMhvLink = true,
-    myVaUpdateErrorsWarnings = true,
-    vaOnlineSchedulingStaticLandingPage = true,
-  } = toggles;
+import FEATURE_FLAG_NAMES from '@department-of-veterans-affairs/platform-utilities/featureFlagNames';
```

Comment:
I remember why I didn't use the export from `@department-of-veterans-affairs/platform-utilities/featureFlagNames` now...

This code actually will not work, and when you try and run the mock server you will get the error:

```SyntaxError: Cannot use import statement outside a module```

This is because you can't use `import` inside the mock serve Node scripts and instead have to use `require()`

Reply:
good catch, thanks! I hadn't run the mock server, only the cypress tests. odd that those passed tho 🤔 . I'll change this to use `require` and test some more with the mock server 👍

Reply:
hmm that might be because the cypress tests run in an es6 module kind of context instead of the Node runtime context.

IDK that you can actually make this work as is since Node is going to complain about that file when you try and require it... let me know though if you can figure it out.

Reply:
hmm that might be because the cypress tests run in an es6 module kind of context instead of the Node runtime context.

IDK that you can actually make this work as is since Node is going to complain about that file when you try and require it... let me know though if you can figure it out.

Reply:
yea, you may be right. I tested a bit with mocker-api, and there isn't a way (afaik) to use an async `import` statement to load platform's ES module that houses the name mapping object, at least within the context of the scripts loaded by the mocker-api cmd line interface locally.

Th only way around this I've found is to use `fs.readFileSync` to load the platform JS file as a string, and then parse out the object, convert to JSON string, and parse it as object.  I have an example of this working, but it i ugly and adds more complexity than I'd like :( ... I'd feel better about it if I knew it was a temp solution :) 

I think we could still use this approach if platform team managed this mapping as a .json file instead of a .js ES module.  If the master list is json, we could more easily support multiple files for different module contexts (e.g. featureFlags.js, fatureFlags.cjs, etc), which means client-side app code and mocking server code would be referencing the same exact list. Not sure if there could be other reasons to avoid doing like this, but would allow for more interoperability and avoid relying on strict naming conventions.

@allisonlu @adamwhitlock1 Maybe we could chat about this at our next pairing hour? Going to put this PR on hold for now 👍

Reply:
I updated this PR with an example of this using a json file for client-to-server flag name mappings.  switching this PR to "in progress" since I'm testing out an idea with some platform changes and I wanted to test this out a bit with CI.  This builds off of changes from earlier, but also supports running mocker-api locally.  Since this changes things at the platform level, we can use this PR to discuss this and/or other approaches for resolving the problem :)